bucket/sudachi.json

```diff
@@ -35,18 +35,14 @@
         "plugins"
     ],
     "checkver": {
-        "url": "https://sudachi.emuplace.app/",
-        "script": [
-            "$jsPath = ((Invoke-WebRequest https://sudachi.emuplace.app/).Content | Select-String -Pattern '/assets/index-\\w+.js').Matches.Value",
-            "$relPath = ((Invoke-WebRequest https://sudachi.emuplace.app$jsPath).Content | Select-String -Pattern '/releases/sudachi-windows-v[\\d.]+.7z').Matches.Value",
-            "return $relPath"
-        ],
-        "regex": "/releases/sudachi-windows-v(?<version>[\\d.]+).7z"
+        "url": "https://api.github.com/repos/emuplace/sudachi.emuplace.app/releases",
+        "jsonpath": "$[?(@.assets[?(@.name =~ /.*windows.*/i)])].tag_name",
```

Comment:
Just a note: this is because apparently they don't always release a Windows binary, so using "latest" would not work.